packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,90 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
```

Comment:
This will mark something as succeeded, where do we print the message? I can't see it inside this function, so might be worth moving it here?

Reply:
Actually, I think we don't need `loader` in this function at all. We can just call `succeed` in the `installDependencies` as we do right now.

Returning from this function w/o error will automatically succeed.

Am I right? Looking at this code, we will succeed if `pod` is available and mark `installing required dependencies` as done.

Reply:
I believe that's right as I already added `loader.succeed()` on `installDependencies` after calling this function.

Great catch!

Reply:
I, however, need the `loader` being passed as arguments because I use it to specify `Installing pods` after `cocoapods` gets successfully installed.

Reply:
I removed the `loader.succeed()` and it doesn't work without it.

The reason is that it keeps updating the loader and even after I use `inquirer.prompt` the loader erases it and gets on top of it, hence the reason to stop it with a `succeed`.

Reply:
Gotcha, so let's maybe add a comment to mention it's needed for `inquirer.prompt` to work.